### ISSUE TITLE
[cont-init.d] 05-changer-uid-gid.sh: exited 267. - Will not start

### ISSUE THREAD

**Issue by four43** [pi-hole/pi-hole#4742]
### Versions
<!-- Please run `pihole -v` and provide the following: -->
(I cannot figure out how to get into your image with a shell. Command and entrypoint changes didn't work.)

Docker Image: 2022.04.3

- Pi-hole:  ???
- AdminLTE:  ???
- FTL:  ???

### Platform
<!-- OS examples: Debian 10, Debian Buster, Ubuntu 18.04 -->
- OS and version:  Raspberry Pi OS on Buster
<!-- Platform examples: Raspberry Pi, Docker, Synology, VM/VPS -->
- Platform:  Raspberry Pi (v1) B+ ancient potato computer, running via Docker Image 2022.04.3

### Expected behavior

Image starts up without errors

### Actual behavior / bug

Not starting and a continuous loop of:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 01-resolver-resolv: applying...       
[fix-attrs.d] 01-resolver-resolv: exited 0.      
[fix-attrs.d] done.                                                                                                                                                                                                                                    
[cont-init.d] executing container initialization scripts...
[cont-init.d] 05-changer-uid-gid.sh: executing... 
[cont-init.d] 05-changer-uid-gid.sh: exited 267.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.                                        
[s6-finish] waiting for services.                                                                                          
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.             
```

### Steps to reproduce

Start Docker container, watch it fail to run. Disk space is not an issue. What's exit 267?

## Debug Token

(will not start)

This seems related perhaps more specifically to https://github.com/pi-hole/docker-pi-hole/blob/master/s6/debian-root/etc/cont-init.d/05-changer-uid-gid.sh

I didn't realize that docker support was an entirely different project with so much in it. That seems strange...

**Comment by four43** [comment]
This is an issue with the docker container specifically it seems. WRONG PROJECT OOPS

**Comment by PromoFaux** [comment]
> I didn't realize that docker support was an entirely different project with so much in it. That seems strange...

>This is an issue with the docker container specifically it seems

I know you've closed this... but I really am unsure what you're talking about here!

**Comment by four43** [comment]
That file I linked to was throwing for some reason. Exit 267 which I couldn't find any docs or references to. I got the little guy fishing on Google because I had 0 results! 

The fix in my case, while using such an old Raspberry Pi was to just downgrade versions. They have this disclaimer:

>>> Due to https://github.com/moby/moby/issues/40734, you may run into issues running 2022.04 and later on host systems with an older version of libseccomp2 ([Such as Debian/Raspbian buster or Ubuntu 20.04](https://pkgs.org/download/libseccomp2), and maybe [CentOS 7](https://pkgs.org/download/libseccomp)).

And even though I'm running a newer version of Raspberry Pi OS, I'm guessing that hardware isn't supporting _something_ properly. The 2022.02.1 Docker image has been much more stable and reliable.

Cheers!
